velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -1562,6 +1563,26 @@ TEST_F(Re2FunctionsTest, split) {
       {"a", "b"},
   });
   assertEqualVectors(expected, result);
+
+  // Test for empty matches
+  result = evaluate("regexp_split(c0, '')", input);
+  expected = makeArrayVector<std::string>({
+      {"", "1", "a", " ", "2", "b", " ", "1", "4", "m", ""},
+      {"", "1", "a", " ", "2", "b", " ", "1", "4", ""},
+      {"", ""},
+      {"", "a", "1", "2", "3", "b", ""},
+  });
+  assertEqualVectors(expected, result);
+
+  // Test for another case of empty matches
+  result = evaluate("regexp_split(c0, '\\s*[a-z]*\\s*')", input);
+  expected = makeArrayVector<std::string>({
```

Comment:
This is the same as in L1569, right? Let's not repeat and make it clear that both patterns produce same results.